Tasks/DotNetCoreCLI/dotnetcore.ts

```diff
@@ -44,6 +44,7 @@ export class dotNetExe {
 
                 var result = dotnet.execSync();
                 if (result.code != 0) {
+                    tl.error(result.stderr.replace("\r\n", "%0D%0A"));
```

Comment:
you should chained replace probably \r with %0D and \n with ... Test it out on Linux agent as well.